### PR TITLE
Protect: Update Firewall Copy

### PR DESCRIPTION
**Pull request by nateweller**
## Proposed changes:

* Update copy in Jetpack Protect's "Firewall" tab in an effort to improve how the features are explained to users. 

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

peb6dq-1eH-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Review Jetpack Protect's "Firewall" tab and validate against the screenshots.

## Screenshots

Screen Shot 2023-07-17 at 20 21 18-fullpage

Screen Shot 2023-07-17 at 20 23 34-fullpage

### CODE REVIEW

**Review by jeherve — DISMISSED** [review]
Changes read well and make sense. I'll let the Marketing team and native english speakers review as well, though :)

**Comment by augurlu** [comment]
> Changes read well and make sense. I'll let the Marketing team and native english speakers review as well, though :)

I believe @robertbpugh has already given a marketing review/approval for this one 👍  peb6dq-1eH-p2#comment-902

**Review by bhoop — DISMISSED** [review]
👍 No technical complaints!

File: projects/plugins/protect/src/js/components/firewall-page/index.jsx [hunk 1]
```diff
@@ -467,15 +467,15 @@ const FirewallPage = () => {
...
+							{ __( 'Enable automatic firewall protection', 'jetpack-protect' ) }
 						</Text>
 						{ ! isSmall && hasRequiredPlan && displayUpgradeBadge && (
 							<span className={ styles.badge }>{ __( 'NOW AVAILABLE', 'jetpack-protect' ) }</span>
 						) }
 					</div>
 					<Text>
 						{ __(
-							'Protect your site against untrusted traffic sources with automatic security rules.',
+							'Block untrusted traffic sources by scanning every request made to your site. Jetpack’s advanced security rules are updated in real-time to protect your site from the latest threats.',
```
Comment (nateweller) [inline projects/plugins/protect/src/js/components/firewall-page/index.jsx]:
```suggestion
							'Block untrusted traffic sources by scanning every request made to your site. Jetpack’s advanced security rules are automatically kept up-to-date to protect your site from the latest threats.',
```

Per a note from the wise @ArSn peb6dq-1eH-p2#comment-908

Removed "updated in real-time" as that is not accurate (yet!).

**Review by bhoop — APPROVED** [review]

**Pull request merged**